apps/crn-server/test/data-providers/reminders.data-provider.test.ts

```diff
@@ -564,6 +565,94 @@ describe('Reminder Data Provider', () => {
       });
     });
 
+    describe('Video upserted in Event Reminder', () => {
+      beforeAll(() => {
+        jest.useFakeTimers('modern');
+      });
+
+      afterAll(() => {
+        jest.useRealTimers();
+      });
+
+      beforeEach(async () => {});
+
+      test('Should fetch the reminder if its video was upserted in the last 24 hours', async () => {
+        const videoRecordingUpdatedAt = '2022-09-01T08:00:00Z';
+        // set the current date to seconds after the video recording updated at
+        const time = DateTime.fromISO(videoRecordingUpdatedAt)
+          .plus({ seconds: 40 })
+          .toJSDate();
+        jest.setSystemTime(time);
+
+        const squidexGraphqlResponse = getSquidexRemindersGraphqlResponse();
+        squidexGraphqlResponse.queryResearchOutputsContents = [];
+        squidexGraphqlResponse.queryEventsContents![0]!.flatData.videoRecordingUpdatedAt =
+          videoRecordingUpdatedAt;
+        squidexGraphqlClientMock.request.mockResolvedValueOnce(
+          squidexGraphqlResponse,
+        );
+
+        const expectedVideoEventUpdatedReminder =
+          getVideoEventUpdatedReminder();
+        expectedVideoEventUpdatedReminder.data.videoRecordingUpdatedAt =
+          videoRecordingUpdatedAt;
+
+        const result = await reminderDataProvider.fetch(fetchRemindersOptions);
+        expect(result).toEqual({
+          total: 1,
+          items: [expectedVideoEventUpdatedReminder],
+        });
+      });
+
+      test('Should not fetch the reminder if its video was upserted after 24 hours', async () => {
```

Comment:
24 hours after what...? I am trying to understand what these timers mean here. 

As far as I can tell we have 2 scenarios to cover:
1. The reminder is available because the video was uploaded anywhere between now and 24 hours ago
2. The reminder is not available because the video was uploaded more than 24 hours ago

There might be a secret 3 scenario in case we dont trust the data which could 
3. The reminder is not available because the video was uploaded in the future (as in - the date of the upload is a future date)

But I don't think we need to cover this. I'm pretty sure all scenarios are covered in this test file somehow but it would be good to make the test descriptions clearer - unless obviously I missed something?

Reply:
Nice! Yeah, I think the test descriptions are not that clear 😅  I added this case 3 in here https://github.com/yldio/asap-hub/blob/CRN-958-reminders-published-event-materials-video/apps/crn-server/test/data-providers/reminders.data-provider.test.ts#L630-L631